### PR TITLE
Addition of reviews to comments about

### PR DESCRIPTION
**Pull request by Aurige**
Addition of reviews to vehicle pooling drivers and passengers
Also adds a missing Transmodel relation from SINGLE JOURNEY to VEHICLE POOLING DRIVER

Answers to [#318](https://github.com/NeTEx-CEN/NeTEx/issues/318)

### CODE REVIEW

File: xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd [hunk 1]
```diff
@@ -368,6 +373,52 @@ Rail transport, Roads and Road transport
...
+		<xsd:sequence>
+			<xsd:element ref="Review" maxOccurs="unbounded"/>
+		</xsd:sequence>
+	</xsd:complexType>
+	<xsd:complexType name="ReviewStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for Review.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="Date" type="xsd:dateTime" minOccurs="0">
```
Comment (skinkie) [inline xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:402]: If it is named a Date why do you use _xsd:dateTime_?

File: xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd [hunk 2]
```diff
@@ -368,6 +373,52 @@ Rail transport, Roads and Road transport
...
+				<xsd:annotation>
+					<xsd:documentation>Date and time of the review</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="Ranking" type="xsd:positiveInteger">
```
Comment (skinkie) [inline xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:407]: Add an explicit range.

File: xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd (hunk 1 above)
Comment (Aurige) [reply xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:402]: right... I'll change it

File: xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd (hunk 2 above)
Comment (Aurige) [reply xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:407]: That's in the annotation below (_most often 1 to 5, and the higher is the better_) . I'm not sure if we can put a constraint on it (there are other ranking mechanisms than the 5 stars), and other Rankings in NeTEx are totally free.

Comment (skinkie) [reply xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:407]: Thinking more in this direction.
```xsd
  <restriction base='integer'>
    <minInclusive value='1'/>
    <maxInclusive value='5'/>
  </restriction>
```

Comment (Aurige) [reply xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:407]: Yes, that can be nice ... but to be agreed by the group and decide if we have to generalise to other Rankings (also for the global ranking some services have a decimal value being the mean of individual rankings).

**Review by skinkie — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Comment by Aurige** [comment]
Part 5 documentation updated

**Pull request merged**